website/aggregated_questions.qmd

```diff
@@ -0,0 +1,251 @@
+
+What percentage of parents in OECD countries will choose to genetically screen their children for cognitive traits [OR other traits] by 20XX?
+
+By what year will more than XX% of the world's population support laws [definition of support laws required] that prevent genetic discrimination in employment and insurance [OR other areas]?
+
+In what year will a global ethical consensus [scope and definition of consensus requirement] on the limits of genetic editing for non-medical enhancements be reached?
+
+By 20XX, will a major global ethics council [scope of council required] have issued guidelines [scope of guideline required] specifically addressing the ethical use of pre-implantation diagnostics for non-medical trait selection?
+
+Will a public opinion poll in 20XX show that over XX% of the population in at least one G7 country [OR other country designation] supports the use of gene-editing for disease prevention in unborn babies [scope of current opinion polls required]?
+
+By 20XX, will there be a documented legal case where parents are sued [scope of lawsuit required] for not using available gene-editing technologies to prevent a hereditary disease in their child?
+
+Will a country pass legislation by 20XX that requires parental licensing [scope of requirement required] before using pre-implantation diagnostics for selecting embryos based on genetic traits?
+
+Will a social movement advocating for "natural birth" [scope of movement required, definition of advocations required] without genetic modifications gain over 1 million followers [scope of support required] on a major social media platform [platform requirement possibly not necessary] by 20XX?
+
+[move to insurance section] By 20XX, will a major insurance company [scope of major required] in the United States offer a reduced premium for families with genetically edited children to resist common genetic diseases?
+
+Will the World Health Organization (WHO) [OR other organization] establish a special division [log of current divisions or sub-organizations required] by 20XX to monitor and advise on ethical issues related to human genetic enhancement?
+
+By 20XX, will a Nobel Peace Prize [OR other award, catalog of awards required] be awarded to an organization that works to ensure equitable access [outcome or effort based] to gene therapy [OR editing] for populations in developing countries [scope of developing nations required]?
+
+By 20XX, will at least one country have established a "genetic enhancement tax" aimed at subsidizing gene-editing or therapy [scope of interventions required] treatments or enhancements for certain populations [scope of populations required]?
+
+By 20XX, will a court in a major jurisdiction [scope of jurisdiction required] rule that children born with genetic enhancements have the right to know the specifics of their genetic modifications?
+
```

Comment:
thought: since the rights of children are legally questionable (and what would a child do with that data?), this should probably have to do with the legal guardians of a child, in cases such as unilateral editing by the mother, adoption, etc.